test/regressions/TestViewer.js

```diff
@@ -30,6 +30,7 @@ function TestViewer(props) {
     document.fonts.addEventListener('loadingdone', handleFontsEvent);
 
     // Use a "real timestamp" so that we see a useful date instead of "00:00"
+    // TODO: uncomment once we enable eslint-plugin-react-compiler eslint-disable-next-line react-compiler/react-compiler
```

Comment:
`useFakeTimers` is not a react hook, it's imported from `sinon` package.

https://github.com/mui/material-ui/blob/79785b79514741486e5ab7d89003ed99ab3470c9/test/regressions/TestViewer.js#L3

Reply:
Can you expand the comment to include why we disable the rule? Also, I think it's clearer if we leave the comment bars

```suggestion
    // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler -- useFakeTimers is not a React hook
```

Reply:
Resolved in this commit https://github.com/mui/material-ui/pull/42626/commits/c7ac34aa90c09a53955f8fcb34549dfd3d4de9f3